compiler/transforms/snax_copy_to_dma.py

```diff
@@ -276,6 +276,10 @@ def match_and_rewrite(self, op: CopyOp, rewriter: PatternRewriter):
         else:
             dma_loop = remaining_strides.pop(0)
 
+        # if lcb is empty, create init lcb with step 1, bound 1
+        if not lcb:
+            lcb = [Stride(1, 1)]
+
```

Comment:
why?

Reply:
if no common contiguous block is found, the list returns empty. The algorithm however tries to access the list (lcb[-1] a couple lines further). The smallest common contiguous block is always one single element. Maybe better to move this change to the lcb algorithm?

Reply:
Yes this makes sense! I think it indeed makes more sense to put it there!